packages/fakta-opptjening-oms/src/components/OpptjeningFaktaForm.spec.tsx

```diff
@@ -3,6 +3,7 @@ import { renderWithIntlAndReduxForm } from '@fpsak-frontend/utils-test/test-util
 import { act, screen } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import React from 'react';
+import type { OpptjeningAktivitetType } from '@k9-sak-web/types/src/opptjening/opptjeningAktivitetType.js';
```

Comment:
Trenger man file extension her?

Reply:
Nei, må ikkje ha det pr no. Meiner eg forsto det slik at ein meiner det er "meir korrekt" å bruke det mtp framtidig kompatibilitet, og eg endra innstillinga i min Intellij til å bruke det når importer blir sett inn automatisk. Derfor det er med her. Men ser ut til å fungere utan og pr no.

Med .js extensions på imports kan feks typescript og nodejs bruke relative imports uten noko meir bundler/byggeverktøy.

https://stackoverflow.com/questions/75807785/why-do-i-need-to-include-js-extension-in-typescript-import-for-custom-module

Reply:
Står litt om det her og: https://dev.to/ayc0/typescript-50-new-mode-bundler-esm-1jic